### PR TITLE
feat(svgUri): add onError prop to SvgUri/Xml/Ast

### PR DESCRIPTION
**Pull request by uqmessias**
# Summary

This solves #1105 and only affects the ts part.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅     |

### CODE REVIEW

**Comment by uqmessias** [comment]
@msand, could you take sometime to review this PR, pls?

**Comment by msand** [comment]
Yeah, sorry for taking time, I've been sick since I took a first quick look on this. Hopefully recovering soon.

**Comment by uqmessias** [comment]
@msand, no worries!
I hope you get well soon :crossed_fingers:

File: src/xml.tsx [hunk 1]
```diff
@@ -101,31 +106,37 @@ export function SvgAst({ ast, override }: AstProps) {
...
     xml,
   ]);
-  return <SvgAst ast={ast} override={override || props} />;
+
+  try {
+    return <SvgAst ast={ast} override={override || props} />;
+  } catch (error) {
+    onError(error);
+    return null;
+  }
```
Comment (msand) [inline src/xml.tsx]: Perhaps it would be more suitable with an Error boundary here?

Comment (uqmessias) [reply src/xml.tsx:122]: @msand, sorry for not taking the time to improve the PR based on your comments. I was planning to do that, but I started some projects and didn't have the time to do it.

**Comment by prashantchothani** [comment]
@msand @uqmessias  Is there anywhere we can check as how the OnError prop can be used to pass a local SVG on Error ? We tried but it is not working

**Comment by uqmessias** [comment]
We're continuing the conversation inside the original issue: #1105

**Pull request merged**